src/snarks/groth16/mpc_utils.cpp

```diff
@@ -0,0 +1,215 @@
+#include "mpc_utils.hpp"
+
+#include "evaluation_from_lagrange.hpp"
+#include "multi_exp.hpp"
+
+#include <libff/algebra/scalar_multiplication/multiexp.hpp>
+
+using namespace libsnark;
+
+namespace libzeth
+{
+
+using ppT = libff::default_ec_pp;
```

Comment:
Remove these `using` directives. They are already specified in the entrypoint (see: `src/test/mpc_test.cpp`). Then, rename file to `.tcc` file and "templatize" the functions and classes.

Reply:
Done in latest.